modules/snooker/src/main/ts/ephox/snooker/resize/Sizes.ts

```diff
@@ -144,5 +150,15 @@ export const setGenericWidth = function (cell: Element, amount: number, unit: st
   Css.set(cell, 'width', amount + unit);
 };
 
+export const getPixelTableWidth = (table: Element<HTMLTableElement>): string => Width.get(table) + 'px';
+export const getPixelTableHeight = (table: Element<HTMLTableElement>): string => Height.get(table) + 'px';
+
+export const getPercentTableWidth = (table: Element<HTMLTableElement>): string => getPercentSize(table, Width.get) + '%';
+export const getPercentTableHeight = (table: Element<HTMLTableElement>): string => getPercentSize(table, Height.get) + '%';
+
+export const isPercentSizing = (table: Element<HTMLTableElement>) => getRawWidth(table).exists((size) => rPercentageBasedSizeRegex.test(size));
```

Comment:
```suggestion
export const isPercentSizing = (table: Element<HTMLTableElement>) => getRawWidth(table).exists(rPercentageBasedSizeRegex.test);
```
Mainly a stylistic thing, but why not do it this way?

Reply:
This will with the `this` property of the call, which can be dangerous for native things.